showcase/gapic-showcase/src/test/java/com/google/showcase/v1beta1/it/ITServerSideStreaming.java

```diff
@@ -98,4 +114,45 @@ public void testGrpc_serverError_receiveErrorAfterLastWordInStream() {
         assertThrows(CancelledException.class, echoResponseIterator::next);
     assertThat(cancelledException.getStatusCode().getCode()).isEqualTo(StatusCode.Code.CANCELLED);
   }
+
+  @Test
+  public void testHttpJson_receiveStreamedContent() {
+    String content = "The rain in Spain stays mainly on the plain!";
+    ServerStream<EchoResponse> responseStream =
+        httpjsonClient
+            .expandCallable()
+            .call(ExpandRequest.newBuilder().setContent(content).build());
+    ArrayList<String> responses = new ArrayList<>();
+    for (EchoResponse response : responseStream) {
+      responses.add(response.getContent());
+    }
+
+    assertThat(responses)
+        .containsExactlyElementsIn(
+            ImmutableList.of(
+                "The", "rain", "in", "Spain", "stays", "mainly", "on", "the", "plain!"))
+        .inOrder();
+  }
+
+  @Ignore(
+      value = "Ignore until https://github.com/googleapis/gapic-showcase/issues/1286 is resolved")
```

Comment:
@lqiu96 : Please also make a note in the issue about this ignored test.

Reply:
Sounds good, will do.